### ISSUE TITLE
Remove usage of deprecated pkg_resources for Python 3.12

### ISSUE THREAD

**Issue by phillipuniverse** [flyingcircusio/pycountry#166]
New DepreactionWarning when running in Python 3.12 with setuptools 68.2.2:

```
test.py:11: in <module>
    import pycountry
../../Library/Caches/pypoetry/virtualenvs/mymodule-zjZyLK7l-py3.12/lib/python3.12/site-packages/pycountry/__init__.py:10: in <module>
    import pkg_resources
../../Library/Caches/pypoetry/virtualenvs/mymodule-zjZyLK7l-py3.12/lib/python3.12/site-packages/pkg_resources/__init__.py:118: in <module>
    warnings.warn(
E   DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
```

**Comment by zware** [comment]
This is mostly a duplicated of #153, but either way it is addressed by #155 (included in #164, duplicated by #162).

**Comment by phillipuniverse** [comment]
Thanks @zware, just (re-)discovered that myself! I'll close this one.
